### PR TITLE
Class1 data

### PR DESCRIPTION
**Pull request by nathanhbsimmons**
added final solutions

### CODE REVIEW

File: 01week/datatypes.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+};
+
+
+//return data type but not NaN
+function dataType(value) {
+  return typeof value
+};
+
+//returns datat type INCLUDING NaN
+typeof
```
Comment (reneemeyer) [inline 01week/datatypes.js]: this would throw an error, why is it here?

File: 01week/datatypes.js [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+  if(arg1 && arg2){
+    return "both arguments are true"
+  } else {
+    return "nope"
+  }
+};
+
+//test if one of two arguments is true
+function oneIsTrue(arg1, arg2){
+  if((arg1 || arg2) && !(arg1 && arg2)){
```
Comment (reneemeyer) [inline 01week/datatypes.js]: the second part `!(arg1 && arg2)` is redundant and not needed

**Review by reneemeyer — CHANGES_REQUESTED** [review]
Nathan, nice job on the PR and the assignment, just a couple things to fix before you merge.

**Review by nathanhbsimmons — COMMENTED** [review]
Suggested changes made

**Pull request merged**